examples/bugs/gh-119/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "gh-119",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint",
+    "build:worker": "pnpm opennextjs-cloudflare",
+    "dev:worker": "wrangler dev --port 8750 --inspector-port 9370",
+    "preview": "pnpm build:worker && pnpm dev:worker",
+    "e2e": "playwright test -c e2e/playwright.config.ts",
+    "cf-typegen": "wrangler types --env-interface CloudflareEnv"
+  },
+  "dependencies": {
+    "next": "15.0.4",
+    "react-dom": "^18.3.1",
+    "react": "^18.3.1"
```

Comment:
is this what makes this reproduce the issue? ideally we should document it somewhere as it is very easy to miss (you can add some "json comments" here 🤔)

Reply:
It's straight from #119 - that's why I make the folder name obvious `gh-119`.

My next PR will follow the same scheme, cloning the repro from the issue.